cloudant-http/src/main/java/com/cloudant/http/internal/interceptors/CookieInterceptor.java

```diff
@@ -47,7 +48,28 @@ public CookieInterceptor(String username, String password, String baseURL) {
             this.auth = String.format("name=%s&password=%s", URLEncoder.encode(username, "UTF-8")
                     , URLEncoder.encode(password, "UTF-8"))
                     .getBytes("UTF-8");
-            ;
+        } catch (UnsupportedEncodingException e) {
```

Comment:
this ctor should just call `this(username, password, baseURL, null)` and avoid duplicating the content.

Reply:
Fixed in a1c95fad4ccf0e845fe4649613fc184c33168220